test/support/mediaSettingsTests.js

```diff
@@ -61,9 +61,17 @@ export function runQualityMenuTests() {
                 .contains('Auto')
                 .click();
 
+            cy.get('[role="menuitem"]').contains('Quality');
```

Comment:
What was error in this case? Why do we need to move this line up?

Reply:
No. There's no specific reason for that. I just moved it when I was updating the test.